src/style.css

```diff
@@ -0,0 +1,80 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.item {
+  height: 300px;
+  flex: 1 0 300px;
+  font-size: 100px;
+  color: white;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.item__1 {
+  background-color: hsl(0, 100%, 50%);
+  order: 1;
+}
+
+.item__2 {
+  background-color: hsl(0, 100%, 40%);
+  order: 2;
+}
+
+.item__3 {
+  background-color: hsl(0, 100%, 30%);
+  order: 3;
+}
+
+.item__4 {
+  background-color: hsl(0, 100%, 20%);
+  order: 4;
+}
+
+.item__5 {
+  background-color: hsl(0, 100%, 10%);
+  order: 5;
+}
+
+.item__6 {
+  background-color: hsl(0, 100%, 0%);
+  order: 6;
+}
```

Comment:
no need to repeat orders if they don't change,
in this case, they already in right order.